client/src/components/ShowListForUser/ShowListForUser.js

```diff
@@ -46,7 +61,23 @@ class ShowListForUser extends Component {
         data: results,
       });
     });
-  }
+  };
+
+  showEditInstanceModal = show => {
+    const {
+      setModalVisibility,
+      createInstanceAndEditShow,
+      selectShow,
+    } = this.props;
+    if (getShowType(show) === 'instance') {
```

Comment:
I added this check to see if an instance was already created. The `createInstanceAndEditShow` function creates an instance from a show series if the instance does not currently exist.